lib/gringotts/gateways/authorize_net.ex

```diff
@@ -692,7 +683,83 @@ defmodule Gringotts.Gateways.AuthorizeNet do
       opts[:config][:mode] == :prod -> @production_url
```

Comment:
can we use case here instead of cond?

Reply:
Infact an `if` is more suitable here. It's always `@test_url` unless it's `:prod`.